pinot-common/src/main/java/org/apache/pinot/common/config/TableConfig.java

```diff
@@ -72,8 +70,9 @@
 
   private TableTaskConfig _taskConfig;
   private RoutingConfig _routingConfig;
-  private Map<InstancePartitionsType, InstanceAssignmentConfig> _instanceAssignmentConfigMap;
+  private QueryConfig _queryConfig;
```

Comment:
Will it be shown as `"QueryConfig": null` in the table config if it's null? 
If yes, might be good to skip printing out this as `"QueryConfig": null`?

Reply:
No, it won't. Check line 225, we only put it when it is not null.

Reply:
Great! Thanks for confirming!